File: ios/Classes/CShieldErrorCode.swift
import Foundation

/// Mirrors CShieldErrorCode.kt (android) — keep both in sync.
enum CShieldErrorCode {
    static let apInvalidSignature  = "ap_invalid_signature"
    static let apSigningFailed     = "ap_signing_failed"
    static let sslNotConfigured    = "ssl_not_configured"
    static let sslPinMismatch      = "ssl_pin_mismatch"
    static let invalidArgument     = "invalid_argument"
    static let nativeError         = "native_error"
}

File: android/src/main/kotlin/com/cmc/c_shield_embedded/c_shield_embedded/CShieldErrorCode.kt
package com.cmc.c_shield_embedded.c_shield_embedded

internal object CShieldErrorCode {
    const val AIP_INVALID_SIGNATURE = "aip_invalid_signature"
    const val AIP_SIGNING_FAILED    = "aip_signing_failed"
    const val SSL_NOT_CONFIGURED    = "ssl_not_configured"
    const val SSL_PIN_MISMATCH      = "ssl_pin_mismatch"
    const val INVALID_ARGUMENT      = "invalid_argument"
    const val NATIVE_ERROR          = "native_error"
}
